docs/README.md

```diff
@@ -65,7 +68,7 @@ app(
 
 ### State
 
-The state describes the data model in your application. In this example, it consists of a single property: `count` which is initialized to 0.
+The state describes the data model in your application. The state must always be an object. In this example, it consists of a single property: `count` which is initialized to 0.
 
```

Comment:
A little punctuation error here: In this example, it of consists of a single property, `count`, which is initialized to 0.

Reply:
I feel like the first comma may be unneeded.